src/_incydr_sdk/actors/client.py

```diff
@@ -198,25 +213,68 @@ def update_actor(
         * **start_date**: `datetime` - The starting date for the user. Accepts a datetime object or a string in the format yyyy-MM-dd (UTC) format. Pass an empty string to clear the field.
         * **end_date**: `datetime` - The departure date for the user.  Accepts a datetime object or a string in the format yyyy-MM-dd (UTC) format.  Pass an empty string to clear the field.
 
-        **Returns**: An [`Actor`][actor-model] object representing the actor.
+        **Returns**: An [`Actor`][actor-modhel] object representing the actor.
         """
-        request_body = {}
-        if notes is not None:
-            request_body["notes"] = None if notes == "" else notes
-        if start_date is not None:
-            request_body["startDate"] = (
-                None if start_date == "" else _create_date(start_date)
-            )
-        if end_date is not None:
-            request_body["endDate"] = None if end_date == "" else _create_date(end_date)
-        try:
-            response = self._parent.session.patch(
-                f"/v1/actors/actor/id/{actor_id}", json=request_body
+        ...
+
+    def update_actor(self, *args, **kwargs):
```

Comment:
Wow, this is clunkier than I thought it would be lol - I haven't written python in a sec.  Functionally looks good though, nice work! We may have to annotate this and the helper methods so they don't appear in the docs. 

Do you think it would be a little cleaner to just check if the first arg is a string or an actor object and then call the appropriate method from there? 

Then we can retain the nicer exception from earlier that was like "`actor_id` provided but none of the others".

Reply:
Another option here would just be to have a single method signature thats like
`update_actor(actor: Actor | string, notes = None, start_date = None, end_date = None) and then check for kwargs if a string is passed, either ignore or throw exception if they pass Actor object and kwargs

Reply:
I like that approach better, I'll switch to that.